dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -53,6 +53,11 @@ RUN pip3 install thrift==0.13.0
 RUN pip2 install libpci
 RUN pip3 install libpci
 
+# The option --no-build-isolation can be removed when upgrading PyYAML to 6.0.1
+# The the PyYAML for python3 has already installed in the base image docker-config-engine
+RUN pip2 install wheel==0.37.1
+RUN pip2 install PyYAML==5.4.1 --no-build-isolation
```

Comment:
>PyYAML [](http://example.com/codeflow?start=17&length=6)

I see it is installed in docker-config-engine-buster, and this Docker is FROM docker-config-engine-buster. Why install again?

Reply:
The python2 is not installed in docker-config-engine-buster, only in docker-platform-monitor and docker-sonic-mgmt-framework.